decimal.go

```diff
@@ -313,6 +313,12 @@ func (d Decimal) Equals(d2 Decimal) bool {
 	return d.Cmp(d2) == 0
 }
 
+// DigitsValue returns the integer that represents the digits of the decimal.
+// This is the value of the decimal scaled by 10^Exponent().
+func (d Decimal) DigitsValue() *big.Int {
```

Comment:
Actually, I thought about it a bit more, and I think it should be called Coefficient().

Coefficient and Exponent are the proper terms for these values. I should have named d.value d.coefficient, but I didn't.

Other proper terms are: significand and mantissa[1]

DigitsValue is not the right name for this.

[1] https://en.wikipedia.org/wiki/Significand